verticapy/machine_learning/vertica/preprocessing.py

```diff
@@ -92,6 +99,97 @@ def Balance(
     -------
     vDataFrame
         vDataFrame of the created view.
+
+    Examples
+    --------
+
+    The following examples provide a basic understanding of usage.
+    For more detailed examples, please refer to the
+    :ref:`user_guide.machine_learning` or the
+    `Examples <https://www.vertica.com/python/examples/>`_
+    section on the website.
+
+    Load data for machine learning
+    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    We import ``verticapy``:
+
+    .. ipython:: python
+
+        import verticapy as vp
+
+    .. hint::
+
+        By assigning an alias to ``verticapy``, we mitigate the risk of code
+        collisions with other libraries. This precaution is necessary
+        because verticapy uses commonly known function names like "average"
+        and "median", which can potentially lead to naming conflicts.
+        The use of an alias ensures that the functions from verticapy are
+        used as intended without interfering with functions from other
+        libraries.
+
+    For this example, we will use the Titanic dataset.
+
+    .. code-block:: python
+
+        import verticapy.datasets as vpd
+
+        data = vpd.load_titanic()
+
+    .. raw:: html
+        :file: SPHINX_DIRECTORY/figures/datasets_loaders_load_titanic.html
+
+    .. ipython:: python
+        :suppress:
+
+        import verticapy.datasets as vpd
+        data = vpd.load_titanic()
+
+    .. note::
+
+        VerticaPy offers a wide range of sample datasets that are
+        ideal for training and testing purposes. You can explore
+        the full list of available datasets in the :ref:`api.datasets`,
+        which provides detailed information on each dataset
+        and how to use them effectively. These datasets are invaluable
+        resources for honing your data analysis and machine learning
+        skills within the VerticaPy environment.
+
+    Function Application
+    ^^^^^^^^^^^^^^^^^^^^^
+
+    First we import the ``Balance`` function:
+
+    .. ipython:: python
+
+        from verticapy.machine_learning.vertica import Balance
+
+    Then we can directly apply it to the dataset:
+
+    .. ipython:: python
+        :okwarning:
+
+        @suppress
+        vp.drop("balance_model")
+
+        Balance(
+            name = "balance_model",
+            input_relation = data,
+            y = "survived",
+            method = "under"
+        )
+
```

Comment:
Please display the created view using a vDataFrame:
```
vp.vDataFrame(name)
```
You can then use HTML representation of the object.